src/plopp/core/typing.py

```diff
@@ -11,3 +18,9 @@ class VisibleDeprecationWarning(UserWarning):
 
 
 VisibleDeprecationWarning.__module__ = 'plopp'
+
+Plottable = Union[VariableLike, ndarray, Node]
+
+PlottableMulti = Union[
+    VariableLike, ndarray, Dict[str, Union[VariableLike, ndarray]], Node
```

Comment:
```suggestion
    VariableLike, ndarray, Dict[str, Union[VariableLike, ndarray, Node]], Node
```
?

Reply:
Can we just have `PlottableMulti = Union[Plottable, Dict[str, Plottable]]`?

Reply:
Yes, good point!